src/addresses.js

```diff
@@ -1,3 +1,21 @@
-module.exports = {
-  KEYS_MANAGER_ADDRESS: '0xfc90125492e58dbfe80c0bfb6a2a759c4f703ca8'
+const local = {
```

Comment:
const `local` is assigned a value but never used

Reply:
good catch!